apps/docs/pages/guides/platform/database-usage.mdx

```diff
@@ -23,20 +23,44 @@ This value is also reported in the [database settings page](https://app.supabase
 ## Database storage management
 
 Supabase uses network-attached storage to balance performance with scalability.
-For Pro and Enterprise projects, disk size expands ~1.5x automatically (e.g., 8GB -> 12GB) when you reach 90% of the disk size.
-Disk size expansion can only occur once every six hours.
-Pro projects can store up to 1024TB.
 
-All projects enter read-only mode when you reach 95% of the disk size. In read-only mode, clients will encounter errors such as `cannot execute INSERT in a read-only transaction`.
+For Pro and Enterprise tier projects, [disk size expands ~1.5x automatically](#paid-tier-automatic-disk-resizing) when you reach 90% of the disk size. Disk size expansion can only occur once every six hours.
```

Comment:
I think there is duplicated content here. We talk about the 1.5x, 90%, 6 hours all below.

Reply:
Yea your right, i've changed it to bullet points to get the main info across with anchor links. Need a rereview if that's ok 😄

Reply:
fixed